app_widgetbook/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: package:very_good_analysis/analysis_options.3.1.0.yaml
+
+linter:
+  rules:
+    - sort_pub_dependencies: false
```

Comment:
I kind of like this option. Do you have a particular reason for disabling this?

Reply:
No particular reason, it's not working either way on my side don't know why 😅. 
I'll remove the line.